python-sdk/example_dags/example_dynamic_task_template.py

```diff
@@ -39,4 +44,15 @@
     ).expand(input_file=get_file_list(path=GCS_BUCKET, conn_id=ASTRO_GCP_CONN_ID))
 
     # [END howto_operator_get_file_list]
+
+    # [START howto_operator_get_value_list]
+    @task
+    def custom_task(row):
+        print(row)
```

Comment:
Do we have a better example, than just printing it?

Reply:
Doing the sum of a table's column if the value is valid float.